cli/src/externalDeviceCommands.js

```
@@ -62,24 +62,24 @@ const externalDevicesCommands = {
         args.slice(1);
-        let host =  await utils.missingTextPrompt('Host', args[1]);
+        let host = await utils.missingTextPrompt('Host', args[1]);
         if (!host) {
             return;
         }
-        let port =  await utils.missingTextPrompt('Port', args[2]);
+        let port = await utils.missingTextPrompt('Port', args[2]);
         if (!port) {
             return;
         }
         if (isNaN(port)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)